pkg/services/apiserver/builder/scheme.go

```diff
@@ -0,0 +1,32 @@
+package builder
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+func ProvideScheme() *runtime.Scheme {
```

Comment:
i think these originally were package level variables because of a pattern copied from `kubernetes/kubernetes` during early PoC branches. do `ProvideScheme` and `ProvideCodecFactory` need to use wire? i'm mostly wondering this because we have multiple API servers with separate schemes (kube-aggregator and data plane aggregator), so it might get confusing if one is provided via wire. it seems like the API server could call these functions directly, but i might be missing something when searching the diff. thoughts?

Reply:
They can absolutely call the functions directly. I hadn't realised there are multiple separate schemes.

I'll do that change :)